shell/platform/android/test/io/flutter/embedding/android/FlutterActivityTest.java

```diff
@@ -391,6 +391,26 @@ public void itLoadsThemedSplashScreenDrawable() throws PackageManager.NameNotFou
     assertNotNull(splashScreen);
   }
 
+  @Test
+  public void itWithMetadataWithoutSplashScreenResourceDoesNotLoadsSplashScreenDrawable()
```

Comment:
is this test name correct?

Reply:
More or less, it has the metadata file but does not have the lines 
```
activityInfo.metaData.putInt(
        FlutterActivityLaunchConfigs.SPLASH_SCREEN_META_DATA_KEY,
        SplashShadowResources.THEMED_SPLASH_DRAWABLE_ID);
```

to put the `<meta-data>`  entry. Updated it to make it a bit clearer though.